rpc/rpcserver/server.go

```diff
@@ -904,31 +904,43 @@ func (s *walletServer) GetTransactions(req *pb.GetTransactionsRequest,
 				"minimum number of recent transactions may not be negative")
 		}
 	}
-
 	_ = minRecentTxs
```

Comment:
I think I know where you were going with the `minimum_recent_transactions` but I think a better pagination strategy is to call with a begin/end window and a max count of transactions (cause then I can get transactions both in increasing/decreasing intervals by adjusting the begin/end indices accordingly).

But if there is another plan for this, let me know so I can rework it.